Source/Core/VideoCommon/NetPlayChatUI.cpp

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoCommon/NetPlayChatUI.h"
+
+#include <imgui.h>
+
+constexpr float DEFAULT_WINDOW_WIDTH = 220.0f;
+constexpr float DEFAULT_WINDOW_HEIGHT = 400.0f;
+
+constexpr size_t MAX_BACKLOG_SIZE = 100;
+
+std::unique_ptr<NetPlayChatUI> g_netplay_chat_ui;
```

Comment:
Does we really need to introduce another global variable/hidden dependency? Having this straddle between the UI and video code is a pretty big hidden dependency. This should probably live within the renderer class and have the necessary methods added to operate with it.

Reply:
We will end up with a dependency either way. Dunno about the Renderer class, wonder what @stenzek things about it...